app/src/main/java/com/netguru/codereview/network/model/ShopListResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+data class ShopListResponse(
+    val list_id: String,
+    val userId: Int,
+    val listName: String
```

Comment:
list_id property is written by snack_case
listName is written by camelCase

Let's use one of the writing type :)